### PR TITLE
Case display for SQL cases

### PR DESCRIPTION
**Pull request by snopoke**
Haven't tested a case with indices or attachments but works for a case without.
@emord 

@dimagi/scale-team FYI

:tropical_fish:

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/case/templatetags/case_tags.py [hunk 1]
```diff
@@ -24,10 +24,182 @@
...
+                        {
+                            "expr": "location_type",
+                            "name": _("Type"),
+                        },
+                        {
+                            "expr": "location_site_code",
+                            "name": _("Code"),
+                        },
+                        #{
+                            #"expr": "last_reported",
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/case/templatetags/case_tags.py]: E131 continuation line unaligned for hanging indent

**Comment by czue** [comment]
fyi not mergeable

**Comment by czue** [comment]
looks good! :+1: when mergeable and :white_check_mark:

**Comment by snopoke** [comment]
Updated with full support for case hierarchy and case attachments and all actions on case and form views

**Comment by snopoke** [comment]
:white_check_mark:

**Pull request merged**